### DISCUSSION TITLE
PID controller implementation

### DISCUSSION THREAD

**Discussion by alexander-thiesen** [WPI-AIM/ambf#211] [Q&A]
Hi @adnanmunawar,
I am using ambf in combination with the surgical_robotics_challenge. 
For my research I need to know how the PID controller for the robot joints is implemented. 
Unfortunately I couldn't find how the PID controller is implemented. 
A explanation or link to its implementation would be apreciated. 

Thank You and best Regards

Alexander Thiesen

**Comment by adnanmunawar** [comment]
Hello Alexander, we had a discussion on this in the surgical_robotics_challegne repo. You can follow it at this link (https://github.com/surgical-robotics-ai/surgical_robotics_challenge/issues/64)
